webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7614,19 +7614,34 @@ var IS_SOCKET_CONNECTED = false;
                 };
                 return conversationDetail;
             };
-            _this.checkBotDetail = function () {	
+            _this.checkBotDetail = function (userId) {	
                 window.Applozic.ALApiService.ajax({	
-                    url: MCK_BOT_API + "/application/" + MCK_APP_ID + "/bot/" + CURRENT_GROUP_DATA.conversationAssignee,	
+                    url: MCK_BOT_API + "/application/" + MCK_APP_ID + "/bot/" + userId,	
                     type: 'get',	
                     global: false,	
                     success: function (data) {	
                                 CURRENT_GROUP_DATA.CHAR_CHECK = data.data[0] && (data.data[0].aiPlatform == KommunicateConstants.BOT_PLATFORM.DIALOGFLOW) && !(data.data[0].autoHumanHandoff);	
+                                if(CURRENT_GROUP_DATA.CHAR_CHECK == false) {
+                                    _this.removeWarningsFromTextBox();
+                                }
```

Comment:
can also be written as `!CURRENT_GROUP_DATA.CHAR_CHECK && _this.removeWarningsFromTextBox()`